vpr/src/place/place.cpp

```diff
@@ -799,13 +752,11 @@ void try_place(const t_placer_opts& placer_opts,
                                       pin_timing_invalidator.get(),
                                       timing_info.get());
 
-        state.t = 0; /* freeze out */
-
         /* Run inner loop again with temperature = 0 so as to accept only swaps
          * which reduce the cost of the placement */
-        placement_inner_loop(state.t, num_temps, state.rlim, placer_opts,
-                             move_lim, state.crit_exponent, quench_recompute_limit, &stats,
-                             &costs, &prev_inverse_costs,
+        placement_inner_loop(&state, placer_opts,
```

Comment:
Could just pass state by reference and clean up the old C-style pointer passing here.